src/helpers/datetime.js

```diff
@@ -0,0 +1,70 @@
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December'
+]
+
+export const convertToDate = (convertedDate, fullMonth) => {
+  const date = new Date(convertedDate);
+  const day = date.getDate();
+  const year = date.getFullYear();
+  if (fullMonth) {
+    const month = months[date.getMonth()];
+    return `${day}. ${month} ${year}`;
+  } else {
+    const month = date.getMonth() + 1;
+    return `${day}. ${month}. ${year}`;
+  }
+}
+
+export const convertToTime = (convertedDate) => {
```

Comment:
Tip: pro nějaké složitější manipulace s datem/časem je docela fajn balíček inspirovaný Carbonem z PHP - [luxon](https://moment.github.io/luxon/#/).